wiki/markdown/Test-Rules.md

```diff
@@ -0,0 +1,280 @@
+            "name": "",
+            "email": "",
+            "id": "",
+            "group": ""
+          },
+          "customData": []
+        },
+        "links": [
+          {
+            "type": "SUBJECT",
+            "target": "df4cdb42-1580-4cff-b97a-4d0faa9b2b22"
+          }
+        ]
+      },
+      {
+        "meta": {
+          "id": "2acd348d-05e6-4945-b441-dc7c1e55534e",
+          "type": "EiffelArtifactPublishedEvent",
+          "version": "1.1.0",
+          "time": 1521452368758,
+          "tags": [],
+          "source": {
+            "domainId": "someDomain",
+            "name": "someName",
+            "serializer": {
+              "groupId": "com.github.Ericsson",
+              "artifactId": "eiffel-remrem-semantics",
+              "version": "0.0.10"
+            }
+          }
+        },
+        "data": {
+          "locations": [
+            {
+              "type": "NEXUS",
+              "uri": "http://host:port/path"
+            }
+          ],
+          "customData": []
+        },
+        "links": [
+          {
+            "type": "ARTIFACT",
+            "target": "df4cdb42-1580-4cff-b97a-4d0faa9b2b22"
+          }
+        ]
+      }
+    ]
+
+
+<h3>Clear All Rules/Clear All Events</h3>
+
+<p>"Clear All" buttons remove all rules respective events. After clicking on button, a pop-up window shows up and asks for confirmation. After confirming all rules or events are removed.</p>
+
+<img src="./images/GUI_TestRules_Clear_All.png"></img>
+
+<h3>Trash Can Button</h3>
+
+<p>It is possible to remove single rule/event by clicking on trash can button next to specific text area.</p>
+
+<h3>Find Aggregated Object</h3>
+
+<p>Clicking on "Find Aggregated Object" button will start the aggregation process. If rules and events are correct, a pop-up window with the aggregated object will show up on the screen.</p>
+
+<img src="./images/GUI_TestRules_Aggregated_Object.png"></img>
+
+<h2>Curl</h2>
+
+<p>It is possible to use curl to get required information. To get information about test rules status, if this functionality is enabled in back end or not, you can execute command below.</p>
+
+    curl -X GET http://<host>:8080/rules/rule-check/testRulePageEnabled?backendurl="http://127.0.0.1:8090/"
+
+<p>To execute rules on specific events with curl, you need to create a JSON file with rules and events. File should contain:</p>
+
+    { 
+        "listEventsJson": [
+            {Event1},
+            {Event2}
+            ...
+        ], 
+        "listRulesJson": [
+            {Rule1},
+            {Rule2},
+            ...
+        ] 
+    }
+    
+<p>And then run curl command below.</p>
+
+    curl -X POST -d "@<path to file>" -H "Content-Type: application/json" http://<host>:8080/rules/rule-check/aggregation?backendurl="http://127.0.0.1:8090/"
+
+---
+**_More information about how to write rules can be found [here](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/Rules.md)._**
```

Comment:
Remember to update this as well if the path is changed

Reply:
Done